src/inflators/text.ts

```diff
@@ -0,0 +1,129 @@
+import { addComponent } from "bitecs";
+import { BackSide, DoubleSide, FrontSide } from "three";
+import { Text as TroikaText } from "troika-three-text";
+import { HubsWorld } from "../app";
+import { TextTag } from "../bit-components";
+import { addObject3DComponent } from "../utils/jsx-entity";
+
+export type TextParams = {
+  value: string;
+  anchorX?: "left" | "center" | "right";
+  anchorY?: "top" | "top-baseline" | "top-cap" | "top-ex" | "middle" | "bottom-baseline" | "bottom";
+  clipRect?: [number, number, number, number] | null;
```

Comment:
does this need to support both null and undefined?

Reply:
Doesn't "?" imply that accepts `undeinfed`?